lib/ImageResize.php

```diff
@@ -267,6 +267,10 @@ public function save($filename, $image_type = null, $quality = null, $permission
                 $background = imagecolorallocate($dest_image, 255, 255, 255);
                 imagefilledrectangle($dest_image, 0, 0, $this->getDestWidth(), $this->getDestHeight(), $background);
             }
+                
+            imagealphablending($dest_image, false);
+            imagesavealpha($dest_image, true);
```

Comment:
Do you have any sample codes and images to prove this change?

Reply:
Hello,
Sure, here is the code used :
```
$image = new \Gumlet\ImageResize($base_image);
$image->resizeToWidth(300);
$image->save('2-1f216112gm37-w300.webp', IMAGETYPE_WEBP);
```

In attachment, 2-1f21g14309151-w300 is the test before the modification, the alpha in the png is not kept.
2-1f216112gm37-w300 is the test after modification.

https://drive.google.com/file/d/1gQIqlC3ZMXtj4JffsRkwyQDrkB99Wa7Y/view?usp=sharing